influxdb/1.2/alpine/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM alpine:3.5
 
 RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
 
+RUN apk add --no-cache curl bash
```

Comment:
Is there a way the script can be made sh compatible? I believe the alpine image has sh and I would prefer not to install bash for an optional feature. Can we also update this script to use the `influx` CLI program too so we can keep `curl` out of this image?